source/math/FloatVectorList.ooc

```diff
@@ -80,6 +59,27 @@ FloatVectorList: class extends VectorList<Float> {
 		}
 	}
 	standardDeviation ::= this variance sqrt()
+	init: func ~default {
+		super()
+	}
+	init: func ~heap (capacity: Int) {
+		super(capacity)
+	}
+	init: func ~fromVectorList (other: VectorList<Float>) {
+		super(other _vector)
+		this _count = other count
+	}
+	init: func ~withValue (capacity: Int, value: Float) {
+		super(capacity)
+		for (i in 0 .. capacity)
+			this add(value)
+	}
+	toVectorList: func -> VectorList<Float> {
+		result := VectorList<Float> new(this count)
+		result _vector = this _vector
+		result _count = this count
+		result
+	}
```

Comment:
Also, why this function? `myVectorList as VectorList<Float>` would do the exact same thing and wouldn't force me to copy everything. Do we ever use it?